.github/workflows/build.yaml

```diff
@@ -0,0 +1,69 @@
+name: build
+
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
@Iqbalabdi for now can we please use another prefix, perhaps `testing-v*` or something like?  Reason being, we are already using `v*` for homebrew as is and I would like to keep this separate for now.